### DISCUSSION TITLE
move cursor with arrow keys

### DISCUSSION THREAD

**Discussion by RustoMCSpit** [linuxmint/discussions#466] [Ideas]
id like a ctrl + m + a (random arbitrary set of keys) to turn on "move cursor with arrow keys mode"

why? in case my touchpad breaks (which it often does) so i can navigate around

moving with numbers could work too in case there are no arrow keys

**Comment by rcalixte** [comment]
As a potential solution for this, there is a setting and an extension available that may suit your needs:

* The setting is under System Settings -> Windows -> Alt-Tab -> Warp mouse pointer to the new focused window.

* The extension is [Adjacent Windows](https://cinnamon-spices.linuxmint.com/extensions/view/99).

Both are options that allow navigation, cursor movement, and or focus switching without the need for a mouse or touchpad.
